testsuite/simulation/modelica/newtonDiagnostics/newtonDiagnostics-01.mos

```diff
@@ -0,0 +1,5644 @@
+// name: newtonDiagnostics-01
+// keywords: nonlinear system start values newton diagnostics
+// status: correct
+//
+// Testing if flag LOG_NEWTON_DIAGNOSTICS is working
+
+loadString("
+  model problem4
+  \"Powell's badly scaled problem\"
+    Real x1(start=10000);
+    Real x2(start=-5);
+  equation
+    0 = 10000*x1*x2-1;
+    0 = exp(-x1) + exp(-x2) - 1.0001;
+  end problem4;
+"); getErrorString();
+
+simulate(problem4, simflags="-newtonDiagnostics -lv=LOG_NLS_NEWTON_DIAG"); getErrorString();
```

Comment:
Tests with log flags are problematic. We test on a wide variety of machines and OS.
How to make sure that none of the values printed below change from one machine to the other one?

You could add a verbose version of the flag `LOG_NLS_NEWTON_DIAG` and print detailed stuff in there and only general stuff that indicates that everything is going well here.

Reply:
And do you need to simulate for a full second or is it enough to initialize and evaluate once? Maybe `stopTime=0.0` would do this.

Reply:
@AnHeuermann the whole point of this feature is to get the diagnostic log. If we don't output that, we will never know if it works properly

Reply:
Agreed that you could set StopTime = 0.0